From 8f2ab3c66fc7c859f8ebb5a0987f5619b9c5d1f1 Mon Sep 17 00:00:00 2001
From: Zidan Wang <zidan.wang@freescale.com>
Date: Thu, 27 Aug 2015 16:49:07 +0800
Subject: [PATCH] MLK-11435 ASoC: fsl_mqs: add snd_soc_pm_ops for mqs machine
 driver

add snd_soc_pm_ops for mqs machine driver to make the trigger
suspend/resume be called in suspend/resume.

Signed-off-by: Zidan Wang <zidan.wang@freescale.com>
---
 sound/soc/fsl/imx-mqs.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/sound/soc/fsl/imx-mqs.c b/sound/soc/fsl/imx-mqs.c
index 4e32e42..3b66799 100644
--- a/sound/soc/fsl/imx-mqs.c
+++ b/sound/soc/fsl/imx-mqs.c
@@ -263,6 +263,7 @@ static struct platform_driver imx_mqs_driver = {
 	.driver = {
 		.name = "imx-mqs",
 		.owner = THIS_MODULE,
+		.pm = &snd_soc_pm_ops,
 		.of_match_table = imx_mqs_dt_ids,
 	},
 	.probe = imx_mqs_probe,
-- 
1.8.0

